proxy/provider.go

```diff
@@ -34,18 +34,28 @@ const (
 
 type Provider interface {
 	/*
-		Returns the Proxy configuration for the given proxy protocol and targetUrl.
+		Returns the Proxy configuration for the given traffic protocol and targetUrl.
 		If none is found, or an error occurs, nil is returned.
 		Params:
-			protocol: The proxy's protocol (i.e. https)
+			protocol: The protocol of traffic the proxy is to be used for. (i.e. http, https, ftp, socks)
 			targetUrl: The URL the proxy is to be used for. (i.e. https://test.endpoint.rapid7.com)
 		Returns:
 			Proxy: A proxy was found.
 			nil: A proxy was not found, or an error occurred.
 	*/
 	Get(protocol string, targetUrl string) Proxy
 	/*
-		Returns the HTTPS Proxy configuration for the given targetUrl.
+		Returns the Proxy configuration for HTTP traffic and the given targetUrl.
+		If none is found, or an error occurs, nil is returned.
+		Params:
+			targetUrl: The URL the proxy is to be used for. (i.e. http://test.endpoint.rapid7.com)
+		Returns:
+			Proxy: A proxy was found.
+			nil: A proxy was not found, or an error occurred.
+	*/
+	GetHTTP(targetUrl string) Proxy
```

Comment:
Can we have a blank line between the comments of every function so that it is easy to recognize the comments of every function are at the top of the function and not at the bottom?